iosHyperskillApp/iosHyperskillApp/Sources/Modules/StepQuiz/Views/Header/StepQuizNameView.swift

```diff
@@ -3,42 +3,11 @@ import SwiftUI
 struct StepQuizNameView: View {
     let text: String
 
-    var dividerLocation = DividerLocation.top
-
-    @Environment(\.colorScheme) private var colorScheme
-
     var body: some View {
-        VStack(alignment: .leading, spacing: LayoutInsets.defaultInset) {
-            if dividerLocation == .top {
-                divider
-            }
-
-            Text(text)
-                .font(.caption)
-                .foregroundColor(.disabledText)
-
-            if dividerLocation == .bottom {
-                divider
-            }
-        }
-    }
-
-    @ViewBuilder
-    var divider: some View {
-        switch colorScheme {
-        case .light:
-            Divider()
-        case .dark:
-            Divider()
-                .overlay(Color(UIColor.separator))
-        @unknown default:
-            Divider()
-        }
-    }
-
-    enum DividerLocation {
-        case top
-        case bottom
+        Text(text)
+            .font(.caption)
+            .foregroundColor(.disabledText)
+            .padding(.top, LayoutInsets.smallInset)
```

Comment:
By Apple's guidelines, it's better to use doubled standard spacing when introducing a new section of the content - in our case, we should just use `.padding(.top)`.

For a problem of the type `code` we need to remove top padding - not equal spacing between "Input / Output Info" and code preview. You can move this padding to the `Appearance` and override this default padding in the `StepQuizCodeView` when rendering `StepQuizNameView`.

<img src="https://user-images.githubusercontent.com/8586063/218940369-43d775e1-a18d-423b-b0e5-f0fcb5800329.png" width="30%">